config.js

```diff
@@ -0,0 +1,19 @@
+const config = {
```

Comment:
This could be a JSON file instead of JS since there's nothing dynamic.

Reply:
@theycallmeswift I think it'd be beneficial to leave it as .js file. JSON doesn't allow comments, and given that we've moved away from a fairly documented framework I think it'd be very beneficial to document certain things, especially the urls and custom_class property. I'd have a terrible time trying to figure out what they do.

Reply:
@LevytsRoman ok, I can understand that argument.  Stick with JS